.github/workflows/review.yaml

```diff
@@ -0,0 +1,22 @@
+name: 'Automagic Review'
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  generate-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
```

Comment:
### security: Ensure that appropriate security practices are followed when handling secrets such as `OPENAI_API_KEY`.  Providing commented documentation or a README on how to set up and securely manage these keys could be beneficial for future contributors.